portal/utils.py

```diff
@@ -11,6 +15,7 @@ def get_invalid_id_popup():
 def get_student_does_not_exist_popup():
     return {'message': "Failed to find student.", 'state': 'danger'}
 
+
```

Comment:
why?

Reply:
There should be 2 empty lines between static functions if you are following the PEP-8 standard which we should be.